src/sickle.ts

```diff
@@ -18,8 +18,8 @@ export interface SickleOutput {
 export function formatDiagnostics(diags: ts.Diagnostic[]): string {
   return diags.map((d) => {
                 let res = ts.DiagnosticCategory[d.category];
-                if (d.file) res += ' at ' + d.file.fileName + ':';
-                if (d.start) {
+                if (d.file) {
+                  res += ' at ' + d.file.fileName + ':';
```

Comment:
This logic now matches the similar code inside of `tsc`.  It's valid to have a start point of 0.